clients/pkg/promtail/positions/positions.go

```diff
@@ -252,12 +253,11 @@ func writePositionFile(filename string, positions map[string]string) error {
 	}
 
 	target := filepath.Clean(filename)
-	temp := target + "-new"
 
-	err = ioutil.WriteFile(temp, buf, os.FileMode(positionFileMode))
+	err = renameio.WriteFile(target, buf, os.FileMode(positionFileMode))
```

Comment:
https://github.com/google/renameio#windows-support

According to this, we should probably special-case Windows here. What do you think?

Reply:
I wanted to reference it. You were faster. I think so. We should use the old version on Windows.